auth-web/src/components/auth/common/HelpDialog.vue

```diff
@@ -1,25 +1,29 @@
 <template>
-  <v-dialog v-model="isDialogOpen" max-width="640">
+  <v-dialog v-model="isDialogOpen" max-width="45rem">
     <v-card>
       <v-card-title>Need Assistance?</v-card-title>
       <v-card-text>
-        <p class="mb-7">If you have not received your Access Letter from BC Registries, or have lost your {{helpDialogFor}},
-          <span>please contact us at:</span>
-        </p>
+        <p class="mb-7" v-html="helpDialogBlurb" />
+
         <ul class="contact-info__list mb-7">
           <li>
-            <span>Toll Free:</span>&nbsp;&nbsp;{{ $t('techSupportTollFree') }}
+            <span>{{ $t('labelTollFree') }}</span>
+            <a :href="`tel:+${$t('maximusSupportTollFree')}`">{{ $t('maximusSupportTollFree') }}</a>
           </li>
           <li>
-            <span>Phone:</span>&nbsp;&nbsp;{{ $t('techSupportPhone') }}
+            <span>{{ $t('labelVictoriaOffice') }}</span>
+            <a :href="`tel:+${$t('maximusSupportPhone')}`">{{ $t('maximusSupportPhone') }}</a>
```

Comment:
Note that local phone number now include the leading "1" as part of the number.

It's not strictly required if you are in Victoria but it's consistent this way. (The en.json file should not specify part of the number, and this template another part of it. Now it all comes from the en.json file.)